packages/shared/src/graphql/sources.ts

```diff
@@ -26,12 +26,17 @@ export enum SourcePermissions {
   WelcomePostEdit = 'welcome_post_edit',
 }
 
+export interface SourceMemberFlag {
+  hideFeedPosts: boolean;
```

Comment:
This one can be `null` on the API so we should make it optional.

Reply:
This one missing `optional` 🙏

Reply:
I don't know what's up with GitHub, but I have actually changed this already.

https://github.com/dailydotdev/apps/pull/1970/files#diff-01cb5b194208871f2275f21d46482164aa75f246128e30dd67acc408efa1f735R29-R30